### PR TITLE
Add docs for deposits and withdrawals through cli

### PR DESCRIPTION
**Pull request by shrenujb**

### CODE REVIEW

File: pages/deposits_and_withdrawals/cli_commands.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+### Deposit to Subaccount
+
+```bash
+dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipient_address> <recipient_subaccount_number> <quantums> [flags]
+```
+
+Example:
+```bash
+dydxprotocold tx sending deposit-to-subaccount dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k 0 <usdc_quantum_uint64> --keyring-backend test --fees 5000adv4tnt
```
Comment (vincentwschau) [inline pages/deposits_and_withdrawals/cli_commands.md:11]: nit: Since the fee denom (`adv4tnt` here) can change based on the network, might be better to preface this with
```
NATIVE_TOKEN_DENOM=adv4tnt
dydxprotocold tx sending deposit-to-subaccount dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k dydx1g2ygh8ufgwwpg5clp2qh3tmcmlewuyt2z6px8k 0 <usdc_quantum_uint64> --keyring-backend test --fees 5000$NATIVE_TOKEN_DENOM
```
or you can even reference the network constants doc [here](https://docs.dydx.exchange/networks/network1/network_constants#native-token-denom) in your doc.

This would make the instructions more chain-agnostic.

Comment (shrenujb) [reply pages/deposits_and_withdrawals/cli_commands.md:11]: Good call

**Review by tqin7 — APPROVED** [review]

**Review by vincentwschau — APPROVED** [review]

File: pages/_meta.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
   "governance": "Governance",
   "security": "Security",
   "terms_and_policies": "Terms and Policies",
+  "deposits_and_withdrawals": "Deposits and Withdrawals",
```
Comment (ttl33) [inline pages/_meta.json:12]: can we move this to be under trading? 

we should keep terms and policies to be last and should group activities that users can do together

File: pages/deposits_and_withdrawals/cli_commands.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+## CLI commands
+
+### Deposit to Subaccount
```
Comment (ttl33) [inline pages/deposits_and_withdrawals/cli_commands.md:3]: here and below: for each section, can you add a short description of each what the actions are?

File: pages/deposits_and_withdrawals/cli_commands.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+```bash
+dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipient_address> <recipient_subaccount_number> <quantums> [flags]
+```
+
+Example:
+```bash
+NATIVE_TOKEN_DENOM=adv4tnt
```
Comment (ttl33) [inline pages/deposits_and_withdrawals/cli_commands.md:11]: We should avoid spelling out the native token denom explicitly here but rather keep all the info in https://docs.dydx.exchange/networks/network1/network_constants#native-token-denom and then only point to that file.

So here, I would suggest removing this line and just have a line saying something like "see this page for the native token denom".

File: pages/deposits_and_withdrawals/cli_commands.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+## CLI commands
+
+### Deposit to Subaccount
+
+```bash
+dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipient_address> <recipient_subaccount_number> <quantums> [flags]
+```
+
+Example:
```
Comment (ttl33) [inline pages/deposits_and_withdrawals/cli_commands.md:9]: also, can we add a line explaining that this is a "self-deposit"?

meaning that this deposits user's USDC from x/bank account to the x/subaccount account?

File: pages/deposits_and_withdrawals/cli_commands.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+### Withdraw from Subaccount
+
+```bash
+dydxprotocold tx sending withdraw-from-subaccount <sender_key_or_address> <sender_subaccount_number> <recipient_address> <quantums> [flags]
+```
+
+Example:
+```bash
+NATIVE_TOKEN_DENOM=adv4tnt
```
Comment (ttl33) [inline pages/deposits_and_withdrawals/cli_commands.md:25]: same as above

File: pages/deposits_and_withdrawals/cli_commands.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+Note: Token denoms for transfer amounts and fees can be found [here](https://docs.dydx.exchange/networks/network1/network_constants#native-token-denom)
+
+
+### Withdraw from Subaccount
+
+```bash
+dydxprotocold tx sending withdraw-from-subaccount <sender_key_or_address> <sender_subaccount_number> <recipient_address> <quantums> [flags]
+```
+
+Example:
```
Comment (ttl33) [inline pages/deposits_and_withdrawals/cli_commands.md:23]: also should explain what this example does in plain english

File: pages/deposits_and_withdrawals/cli_commands.md (hunk 3 above)
Comment (ttl33) [reply pages/deposits_and_withdrawals/cli_commands.md:3]: for example, this allows depositing into subaccount

File: pages/deposits_and_withdrawals/cli_commands.md (hunk 4 above)
Comment (shrenujb) [reply pages/deposits_and_withdrawals/cli_commands.md:11]: The NATIVE_TOKEN_DENOM var is just for ease of use on the cmd here. I already have the note pointing to the url

File: pages/deposits_and_withdrawals/cli_commands.md (hunk 5 above)
Comment (shrenujb) [reply pages/deposits_and_withdrawals/cli_commands.md:9]: This doesn't need to be a self deposit. You can deposit funds to a different address subaccount

File: pages/_meta.json (hunk 2 above)
Comment (shrenujb) [reply pages/_meta.json:12]: Done
